### ISSUE TITLE
Merge cells and apply cell coloring to full colspan?

### ISSUE THREAD

**Issue by Horsmann** [gitbrent/PptxGenJS#680]
Hi, 
I am trying to let a row span over three cols - this works but my cell coloring is only applied to the first cell, the other two seem to fallback to the default cell coloring.
Can I somehow combine a custom cell border coloring with colspan?

Here my code snippet how I am trying to do that currently:
```
	var arrBorder = [ 
					  {color: '0000FF',pt: 2}, 
					  {color: 'FFFFFF',pt: 2}, 
					  {color: '0000FF',pt: 2}, 
					  {color: 'FFFFFF',pt: 2},
					];

	rows.push([
		{text: name.toUpperCase(),
			options: {bold: true, fontSize: fs, fontFace: ff,
					  border: arrBorder,
					  colspan: 3 
				      }},
```

**Comment by gitbrent** [comment]
Have you looked through all sample code under /demos?

<img width="1599" alt="Screen Shot 2020-02-25 at 21 01 06" src="https://user-images.githubusercontent.com/7218970/75307696-fe28da80-5811-11ea-968c-5c6d46719c1c.png">

**Comment by Horsmann** [comment]
Yes, I actually have :). The examples you have there don't use the cell border coloring in combination with ´colspan´. They do just colspan with fill-color (this seems to work). 
The array boundary colors seem to be applied to the first cell only if used in combination with `colspan`, or what what do you mean?
The code for the slide you posted is this one:

´´´
		var tabOpts2 = { x:0.5, y:3.3, w:12.4, h:1.5, fontSize:14, fontFace:'Courier', align:'center', valign:'middle', fill:'F9F9F9', border:{pt:'1',color:'c7c7c7'}};
		var arrTabRows2 = [
			[
				{ text:'A1\n--\nA2', options:{rowspan:2, fill:'99FFCC'} },
				{ text:'B1\n--\nB2', options:{rowspan:2, fill:'99FFCC'} },
				{ text:'C1 -> D1',   options:{colspan:2, fill:'9999FF'} },
				{ text:'E1 -> F1',   options:{colspan:2, fill:'9999FF'} },
				'G1'
			],
			[ 'C2','D2','E2','F2','G2' ]
		];
		slide.addTable( arrTabRows2, tabOpts2 );
´´´

**Comment by Horsmann** [comment]
btw. I circumvented my problem by letting the row below/above color the merged cell. This wouldn't work if they were merged cells, too.

**Comment by pandersen-uncharted** [comment]
This is still happening. If I have a table cell with a colspan > 1 and a bottom border the border is only rendered under the first cell of the span. Unfortunately I can't use the workaround of setting the top border of the following row since that also has multi-column spanning cells.

**Comment by gitbrent** [comment]
Can someone please provide a test case that will reproduce the issue?

**Comment by pandersen-uncharted** [comment]
Here's a quick and dirty example. (I just inserted this into your jsFiddle.)

```javascript
let cells = [];
cells.push( {
	text: 'Spanning header',
	options: {
		color: 000000,
		bold: true,
		align: 'center',
		border: [
			{ pt: 0, color: 000000 },
			{ pt: 0, color: 000000 },
			{ pt: 2, color: 000000 },
			{ pt: 0, color: 000000 },
		],
		colspan: 2
	}
} );
let rows = [];
rows.push(cells);
cells = [];

cells.push( {
	text: 'Content 1',
	options: {
		color: 000000,
		border: [
			{ pt: 0, color: 000000 },
			{ pt: 0, color: 000000 },
			{ pt: 0, color: 000000 },
			{ pt: 0, color: 000000 },
		],
		colspan: 1
	}
} );

cells.push( {
	text: 'Content 2',
	options: {
		color: 000000,
		border: [
			{ pt: 0, color: 000000 },
			{ pt: 0, color: 000000 },
			{ pt: 0, color: 000000 },
			{ pt: 0, color: 000000 },
		],
		colspan: 1
	}
} );
rows.push(cells);

let tableOptions = {
	x: 0,
   	y: 0,
	w: 3,
	h: 2
};

slide.addTable(rows, tableOptions);

```
And this is the result. Notice the border is only on the first half of the spanning cell.
<img width="365" alt="Screen Shot 2020-06-03 at 7 59 08 AM" src="https://user-images.githubusercontent.com/28305358/83634295-46be0580-a570-11ea-9853-8c197ea29f1a.png">
